db/migrations/Reference.php

```diff
@@ -33,5 +35,7 @@ protected function references(Blueprint $table, $targetTable, $fromColumn, $setP
             ->references('id')->on($targetTable)
             ->onUpdate('cascade')
             ->onDelete('cascade');
+
+        return $col;
```

Comment:
I'm returning the column definition here.  
For example the question answerer field should be nullable.